### PR TITLE
Updates for pypi and test cases

### PR DESCRIPTION
**Pull request by max-zilla**
Adds proper test case for canopy cover function, and moves logic to coerce CC value to 0-100 scale instead of 0-1 here instead of in extractor.

### CODE REVIEW

**Comment by max-zilla** [comment]
Updated to include pypi logic and terraref shared namespace.

```
import terraref.stereo_rgb

from terraref.stereo_rgb import *

from terraref.stereo_rgb import calculate_canopycover
```

**Comment by max-zilla** [comment]
update to use RST instead of MD

**Review by ZongyangLi — APPROVED** [review]
It looks good to me.

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+
+setup(name='terraref-stereo_rgb',
+      version='1.0.4',
+      description='TERRA-REF stereo RGB camera science package',
+      long_description=description(),
+      packages=find_packages(),
+      namespace_packages=['terraref'],
       include_package_data=True,
       url='https://github.com/terraref/stereo_rgb',
+      install_requires=[
```
Comment (dlebauer) [inline setup.py:17]: Add classifiers and keywords sections? Should we standardize these, or is there some hierarchy where this is covered in the terraref namespace?

e.g. 

keywords=['field crop', 'phenomics', 'computer vision', 'remote sensing']
'Topic :: Scientific/Engineering :: GIS',

**Review by dlebauer — APPROVED** [review]
Where is all of the stitching done? Have the caveats and documentation deleted from the README here been moved there?

For functionality that we plan to add to this package (e.g. texture analysis?) could we leave the text in the README, but commented out?

File: setup.py (hunk 1 above)
Comment (max-zilla) [reply setup.py:17]: added keywords and classifiers, not standardized at the moment

**Comment by max-zilla** [comment]
stitching is still done in fieldmosaic code in extractors-stereo-rgb repo but we should find a better place as the code isnt specific to stereo-rgb. caveats are there.

**Pull request merged**